dl/src/main/scala/com/intel/analytics/bigdl/models/fasterrcnn/Visualizer.scala

```diff
@@ -0,0 +1,115 @@
+/*
+ * Licensed to Intel Corporation under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * Intel Corporation licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.intel.analytics.bigdl.models.fasterrcnn
+
+/*
+ * Licensed to Intel Corporation under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * Intel Corporation licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import java.awt.image.BufferedImage
+import java.awt.{BasicStroke, Color, Font, Graphics2D}
+import java.io.File
+import java.nio.file.Paths
+import javax.imageio.ImageIO
+
+import com.intel.analytics.bigdl.tensor.Tensor
+import org.apache.log4j.Logger
+
+/**
+ * visualize detections to image
+ */
+object Visualizer {
+  val logger = Logger.getLogger(getClass)
+  val bgColor = new Color(0, 0, 1, 127)
```

Comment:
yes, but it may not be necessary if just for demo, 
code in original fasterrcnn is also hard coded